cypress/page/shoping-cart.page.ts

```diff
@@ -2,7 +2,7 @@ class ShopingCartPage {
   private shoppingMenu: string;
 
   constructor() {
-    this.shoppingMenu = ".cart_navigation span";
+    this.shoppingMenu = '.cart_navigation a[title="Proceed to checkout"]';
```

Comment:
probably just `a[title="Proceed to checkout"]` can work

Reply:
resolved

Reply:
![image](https://user-images.githubusercontent.com/33491290/181925275-998f693f-c2f4-4e78-a400-10a4e2673010.png)